config/dashboards/dashboards_config.go

```diff
@@ -172,7 +172,7 @@ func unmarshal(yamlString string) (out MonitoringDashboardsList, err error) {
 	list := new(MonitoringDashboardsList)
 	err = yaml.Unmarshal([]byte(yamlString), &list)
 	if err != nil {
-		return nil, fmt.Errorf("failed to parse monitoring dashboard yaml data. error=%v", err)
+		return nil, fmt.Errorf("failed to parse monitoring dashboard yaml data. error=%w", err)
```

Comment:
Was there a reason to return the wrapped error now?

Reply:
considered best practice in Go to use %w in Errorf. I changed a bunch of things so we get the full wrapped error for better debugging if/when needed. Even Kubernetes [is trying to move to this](https://github.com/kubernetes/kubernetes/issues/123234)